yarn.lock

```diff
@@ -7332,7 +7332,7 @@ globby@11.1.0, globby@^11.1.0:
     merge2 "^1.4.1"
     slash "^3.0.0"
 
-globby@13.1.4, globby@^13.1.1, globby@^13.1.4:
+globby@13.1.4:
```

Comment:
It would be nice to wait for `markdownlint-cli2` release in order to avoid the need for two `globby` packages, due to their locked version requirement 🙈

Reply:
Looks like `markdownlint-cli2` has released [a new version](https://github.com/DavidAnson/markdownlint-cli2/blob/main/CHANGELOG.md), the bump for that should be coming with the next dependabot wave, sadly, the `globby` package version is still left hard-coded. 🙈